cms/templates/js/course-outline.underscore

```diff
@@ -57,11 +91,42 @@ if (staffOnlyMessage) {
         addStatusMessage(messageType, messageText);
     }
 
-    if (hasPartitionGroups) {
+    if (hasPartitionGroups && numAllowedGroups === 0 && xblockInfo.isVertical()) {
         addStatusMessage(
             'partition-groups',
             gettext('Access to some content in this unit is restricted to specific groups of learners')
         );
+    } else if (numAllowedGroups >= 1 && xblockInfo.isVertical()) {
+        var selectedGroupsLabel = '';
+
+        for (var schemeId in groupAccessMap) {
+            var groups = groupAccessMap[schemeId];
+            for (var i = 0; i < groups.length; i++) {
+                var currentGroup = idToName[schemeId][groups[i]];
+
+                if (selectedGroupsLabel.length === 0) {
+                    selectedGroupsLabel = currentGroup;
+                } else {
+                    // Translators: This is building up a list of groups. It is marked for translation because of the
```

Comment:
I don't think this string will get through to the translators because it is not directly above the gettext call. Read through all the comments about Translator comments in http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#multi-line-strings.

In general, getting translator comments through in JavaScript files can be quite tricky. To actually be sure that it is working, you probably would need to generate translation files locally.